curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/6552303a9a78704f8ff072e9.md

```diff
@@ -7,10 +7,14 @@ dashedName: step-40
 
 # --description--
 
+Now it is time to test out the pause button.
+
 You need to hook up the `pauseSong` function to an event listener to make it work.
 
 Add a `click` event listener to the `pauseButton` element, then pass in `pauseSong` as the second argument of the event listener. This is the function the event listener will run.
```

Comment:
```suggestion
Add a `click` event listener to the `pauseButton` element, then pass in `pauseSong` as the second argument of the event listener. This is the function the event listener will run.
```
From my understanding of the related issue, original first sentence should be removed.

Reply:
@gikf Thank you for the review! 

I missed it the first time, my bad 😅